src/main/webapp/js/author_search.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function(){
+
+    // config variables
+    let solr_url = '../solr/ubo/select/';
```

Comment:
use mycore solr servlet URL here (without trailing '/'):
'../servlets/solr/select'
so it always works.